packages/css/js/Css_Js_Core.ml

```diff
@@ -31,7 +31,7 @@ module type MakeResult = sig
 
   val insertRule : string -> unit
   val renderRule : renderer -> string -> unit
-  val global : string -> rule array -> unit
+  val global : rule array -> unit
```

Comment:
Minor: How about if we use `list` instead?

Reply:
in CssJs we use arrays everywhere (and moving forward we will get rid of Css which uses lists).

Why do you suggest list here?

Reply:
No other reason apart from the imperative nature of arrays over lists.

Reply:
I think this follows the `rule` variant `S`. In Css_Js_Core, `S` is defined `string * rule array` and on Css_Legacy_Core it's `string * rule list`